src/App.vue

```diff
@@ -280,8 +280,8 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
     }
   }
 
-  @Watch('stateModel.nameRequest.entityType')
-  private onEntityTypeChanged (val: string | null) : void {
+  @Watch('state.nameRequest.entityType')
```

Comment:
I'm not sure this is correct. I had to change the reference to the `stateModel` which we had bound to `App`, as there is no `state` property bound here.

Reply:
Oops, merge issue. Fixing now.

Reply:
I found another place where my rebase restored some deleted whitespace :/

Reply:
excellent!